### PR TITLE
[BABEL-1305] - change Cancellation entity to accept Identifier identi…

### PR DESCRIPTION
**Pull request by lazaki**
change Cancellation entity to accept Identifier instead of contractId

### CODE REVIEW

File: src/main/java/ch/baloise/corellia/api/entities/Cancellation.java [hunk 1]
```diff
@@ -26,13 +22,9 @@
   @JsonPropertyDescription("A code uniquely identifying the cancellation reason")
   private Integer reasonCode;
```
Comment (dweber019) [inline src/main/java/ch/baloise/corellia/api/entities/Cancellation.java:23]: We had a talk with Busy and they now need the action type too. I would suggest to reuse the `Action ` type and drop the `effectiveDate` and use the one in `Action`. The UGVS service will be changed by @schlijo

Comment (lazaki) [reply src/main/java/ch/baloise/corellia/api/entities/Cancellation.java]: Changed :+1:

File: src/main/java/ch/baloise/corellia/api/entities/Cancellation.java [hunk 2]
```diff
@@ -1,52 +1,47 @@
...
+    @NotNull
+    @Valid
+    private Action action;
 
-  @NotNull
-  @JsonPropertyDescription("A code uniquely identifying the cancellation reason")
-  private Integer reasonCode;
+    @NotNull
+    @JsonPropertyDescription("A code uniquely identifying the cancellation reason")
+    private Integer reasonCode;
```
Comment (dweber019) [inline src/main/java/ch/baloise/corellia/api/entities/Cancellation.java:22]: The reason code is now in `Action` you can remove this.

Comment (lazaki) [reply src/main/java/ch/baloise/corellia/api/entities/Cancellation.java:22]: :+1:

**Comment by schlijo** [comment]
Well done.

**Pull request merged**